### PR TITLE
Add guide ansible_tower from ansible.redhatgov.io

### PR DESCRIPTION
**Pull request by liquidat**
As discussed, this PR adds the Tower guide from ansible.redhatgov.io.

### CODE REVIEW

File: guides/ansible_tower/1-install/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+---
+**NOTE:** Installation will take some time
+Step 7 will take approx. 10-15 minutes to complete.  This may be a good time to take a break.
+
+---
+
+### End Result
+
+At this point, your Ansible Tower installation should be complete. You can access your Tower (not forgetting that *workshopname* is the name of your workshop, and *#* is your student number) at:
```
Comment (tima) [inline guides/ansible_tower/1-install/README.md]: I don't think this is necessarily true for all uses of lightbulb and a specific detail of how redhatgov.io executes workshops.

File: guides/ansible_tower/1-install/README.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+```bash
+https://workshopname.tower.#.redhatgov.io
```
Comment (tima) [inline guides/ansible_tower/1-install/README.md]: Same as line 93.

File: guides/ansible_tower/2-config/README.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
+# Exercise 2 - Configuring Ansible Tower
+
+In this exercise, we are going to configure Tower so that we can run a playbook.
+
+## The Tower UI
+
+There are a number of contructs in the Ansible Tower UI that enable multi-tenancy, notifications, scheduling, etc. However, we are only going to focus on a few of the key contructs that are required for this workshop today.
```
Comment (tima) [inline guides/ansible_tower/2-config/README.md]: @dfederlein I don't think constructs is our standardized language typically. What does your team use? Concepts?

File: guides/ansible_tower/2-config/README.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+![Ansible Tower Login Screen](ansible-lab-figure01-logon-screen.png)
+
+As soon as you login, you will prompted to request a license or browse for an existing license file
+
+![Uploading a License](at_lic_prompt.png)
+
+### Step 2
+
+In a seperate browser tab, browse to [https://www.ansible.com/workshop-license](https://www.ansible.com/workshop-license) to request a workshop license.
```
Comment (tima) [inline guides/ansible_tower/2-config/README.md]: Please add what a workshop license is -- 5 nodes for X days. (Can't recall off the top of my head.) 

Rhetorical question: should we mention and add a link to the longer-term free trial license also?

File: guides/ansible_tower/3-create/README.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+http://<IP_of_node-1_or_node-2>
+```
+
+If all went well, you should see something like this, but with your own custom message:
+
+![New Website with Personalized Test Message](at_web_tm.png)
+
+## End Result
+
+At this point in the workshop, you've experienced the core functionality of Ansible Tower.  But wait... there's more! You've just begun to explore the possibilities of Ansible Core and Tower.  Take a look at the resources page in this guide to explore some more features.
```
Comment (tima) [inline guides/ansible_tower/3-create/README.md]: Please remove "But wait... there's more!"

File: guides/ansible_tower/2-config/README.md (hunk 3 above)
Comment (liquidat) [reply guides/ansible_tower/2-config/README.md]: In the documentation when we speak about playbooks and tags, it is called "concepts". See for example here: http://docs.ansible.com/ansible-tower/latest/html/userguide/

**Comment by liquidat** [comment]
All requested changes were performed. Should be ok now.

**Comment by tima** [comment]
Thanks @liquidat.

**Pull request merged**